test/cloudant.test.js

```diff
@@ -70,41 +70,6 @@ describe('cloudant connector', function() {
     });
   });
 
-  describe('updateAll and updateAttributes', function() {
```

Comment:
Why are these tests being removed? If the functionality they are testing no longer works with these tests, doesn't that mean this is a breaking change?